src/components/TodoList/TodoList.tsx

```diff
@@ -25,41 +63,32 @@ export const TodoList: React.FC = () => {
         </thead>
 
         <tbody>
-          <tr data-cy="todo">
-            <td className="is-vcentered">1</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">delectus aut autem</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">2</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">quis ut nam facilis et officia qui</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo" className="has-background-info-light">
+          {filteredTodos.map(todo => (
+            <tr data-cy="todo">
+              <td className="is-vcentered">{todo.id}</td>
+              <td className="is-vcentered">
+                {todo.completed && (
+                  <span className="icon" data-cy="iconCompleted">
+                    <i className="fas fa-check" />
+                  </span>
+                )}
+              </td>
+
+              <td className="is-vcentered is-expanded">
+                <p className={!todo.completed ? 'has-text-danger' : 'has-text-success'}>{todo.title}</p>
+              </td>
+
+              <td className="has-text-right is-vcentered">
+                <button data-cy="selectButton" className="button" type="button" onClick={() => setCurrentTodo(todo)}>
+                  <span className="icon">
+                    {currentTodo?.id === todo.id ? <i className="far fa-eye-slash" /> : <i className="far fa-eye" />}
```

Comment:
why not just set different classNames instead of rendering different elements? Use classnames library for it